### PR TITLE
fix(IAM Identity): re-gen with latest API definition

### PR DESCRIPTION
**Pull request by Somchavan**
## PR summary
Java SDK doc for IAM identity. PR includes changes related
1. Create mfa report
2. Get mfa report
3. Get mfa user status
4.  Change token expiration max and default value to 259200

### CODE REVIEW

**Comment by padamstx** [comment]
@Somchavan I noticed that the travis build failed for this PR due to the "main" method in the IamIdentityExamples class being too long (checkstyle error).  So, I just modified the checkstyle config to increase the limit and pushed that to your branch.

File: modules/iam-identity/src/main/java/com/ibm/cloud/platform_services/iam_identity/v1/IamIdentity.java [hunk 1]
```diff
@@ -91,7 +96,7 @@ public class IamIdentity extends BaseService {
   /**
    * Default service endpoint URL.
    */
-  public static final String DEFAULT_SERVICE_URL = "https://iam.cloud.ibm.com";
+  public static final String DEFAULT_SERVICE_URL = "https://iam.test.cloud.ibm.com";
```
Comment (padamstx) [inline modules/iam-identity/src/main/java/com/ibm/cloud/platform_services/iam_identity/v1/IamIdentity.java:99]: This change should be reverted.  We should always use the production endpoint URL in the generated code.  My guess is that you used the test branch version of the API definition when generating the code, and you should be using the production version.

**Review by padamstx — CHANGES_REQUESTED** [review]
The default service endpoint address should reflect the production instance of the service.

**Review by padamstx — APPROVED** [review]
LGTM, just had the one comment about the "if" statement in the integration test class.

File: modules/iam-identity/src/main/java/com/ibm/cloud/platform_services/iam_identity/v1/IamIdentity.java (hunk 1 above)
Comment (Somchavan) [reply modules/iam-identity/src/main/java/com/ibm/cloud/platform_services/iam_identity/v1/IamIdentity.java]: this is fixed.

File: modules/iam-identity/src/test/java/com/ibm/cloud/platform_services/iam_identity/v1/IamIdentityIT.java [hunk 2]
```diff
@@ -1580,32 +1586,35 @@ public void testCreateReport() throws Exception {
...
     public void testGetReportIncomplete() throws Exception {
-        try{
+        try {
+        	System.out.println("reference for report get: "+reportReference );
             GetReportOptions getReportOptions = new GetReportOptions.Builder()
                     .accountId(ACCOUNT_ID)
                     .reference(reportReference)
                     .build();
             Response<Report> response = service.getReport(getReportOptions).execute();
+            if(response.getStatusCode() == 204) {
```
[the remark this answers is not in the document]
Comment (Somchavan) [reply modules/iam-identity/src/test/java/com/ibm/cloud/platform_services/iam_identity/v1/IamIdentityIT.java]: this is fixed.

**Pull request merged**